Marlin/src/HAL/LPC1768/inc/SanityCheck.h

```diff
@@ -31,7 +31,11 @@
 /**
  * Detect an old pins file by checking for old ADC pins values.
  */
-#define _OLD_TEMP_PIN(P) PIN_EXISTS(P) && _CAT(P,_PIN) <= 7 && _CAT(P,_PIN) != 2 && _CAT(P,_PIN) != 3
+#if LPC1768_IS_SKRV1_3
```

Comment:
Why other LPC boards can't use those pins?

If is something specific for a board, maybe it should be defined in the variant, or in the board.

Reply:
They do I checked and the P0_00 and P0_01 and used on the SKR V1.4 board as the I2C signal pins.  But the datasheet on the  LPC series says that those pins are general I/O pins.  I have no idea why @p3p limited it on the SanityCheck.h file. @p3p Did do it right when he defined the pins for the LPC framework.  But your right I can get rid of the "LPC1768_IS_SKRV1_3" directive and just change the line in SanityCheck.h file I will do that now.

Reply:
Fixed this